wordpress-mobile/WordPress-Android
# Issue/6851 add epilogue social

### Fix
Add an epilogue screen for social signup as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6851.

### Test
0. Clear app data.
1. Tap ***Sign Up*** button.
2. Tap ***Sign Up with Google*** button.
3. Choose Google account.
4. Notice ***Epilogue for Social*** screen.
5. Edit ***Display Name*** field.
6. Notice display name in header updates while editing.
7. Tap ***Continue*** button.
  a. If display name was changed, notice progress dialog.
  b. If display name was not changed, notice no progress dialog.

#### Note
To test this code with a `wasabiDebug` or `wasabiRelease` build, you must go through the steps to add your development environment configuration to the `google-services.json` file.  Alternatively, you can download a `vanillaDebug`, `vanillaRelease`, `zaplhaDebug`, or `zalphaRelease` APK from [Buddybuild](https://dashboard.buddybuild.com/apps/595de4daf7f33600018ba24f/build/5a455c2d210023000173a836).
